src/org/openstreetmap/josm/gui/tagging/presets/TaggingPresetSelector.java

```diff
@@ -199,8 +201,9 @@ public String toString() {
      * Constructs a new {@code TaggingPresetSelector}.
      * @param displayOnlyApplicable if {@code true} display "Show only applicable to selection" checkbox
      * @param displaySearchInTags if {@code true} display "Search in tags" checkbox
+     * @param displayDeprecated if {@code true} display "Applicable in region" checkbox
      */
-    public TaggingPresetSelector(boolean displayOnlyApplicable, boolean displaySearchInTags) {
+    public TaggingPresetSelector(boolean displayOnlyApplicable, boolean displaySearchInTags, boolean displayDeprecated) {
```

Comment:
Should we add a checkbox for regions as well?

Reply:
We could, yes. Realistically, the "primary" caller will probably just use `DisplayCheckboxes.values()` instead of hardcoding the current values.

Reply:
> Realistically, the "primary" caller will probably just use `DisplayCheckboxes.values()` instead of hardcoding the current values.

Does that mean we would use the preference for the checkbox associated with the filter like `ONLY_APPLICABLE.getPref()` instead of using `bool` value? (getPref will return boolean value)